### ISSUE TITLE
Comment notification issues

### ISSUE THREAD

**Issue by zorbathut** [themotte/rDrama#476]
See https://www.themotte.org/post/216/meta-something-shiny-and-two-things/55527?context=8#context

tl;dr: Something is wrong with comment notification ordering. It seems to be related to getting multiple responses to one comment without checking them, but it results in comment notifications being buried. I've seen this personally and I've seen multiple reports on it.

It is also probably true that we should change how notifications work; right now you get a notification with the comment *you* posted, which is awkward if you're getting multiple replies, and we should probably get one notification per comment reply, either with *just* the reply (a la Reddit) or with a button to push to see the parent comment.

It's unclear if we should be trying to fix this or trying to redesign how the notification works. Ideas welcome, but we really do need to fix this one way or another.
